src/pageObject.js

```diff
@@ -1,15 +1,34 @@
 'use strict';
 
 class Header {
-
+  logo = 'Conduit logo';
 };
 
 class PageObject {
+  constructor(url, header) {
```

Comment:
The constructor of the `PageObject` class should only take a `url` parameter. Remove the `header` parameter. 🚀